stubs/pyflakes/pyflakes/scripts/pyflakes.pyi

```diff
@@ -0,0 +1,2 @@
+__all__ = ["check", "checkPath", "checkRecursive", "iterSourceCode", "main"]
+from pyflakes.api import check, checkPath, checkRecursive, iterSourceCode, main
```

Comment:
Please note that type checkers don't re-export imports unless using `as`:

```suggestion
from pyflakes.api import check as check, checkPath as checkPath, checkRecursive as checkRecursive, iterSourceCode as iterSourceCode, main as main
```

Reply:
Thanks :)